src/Groups/types.jl

```diff
@@ -142,6 +142,11 @@ It is displayed as product of disjoint cycles.
 """
 const PermGroupElem = BasicGAPGroupElem{PermGroup}
 
+function Base.hash(x::PermGroupElem, h::UInt)
+  b = UInt(GAPWrap.HashPermutation(GapObj(x)))
+  return xor(h, b)
```

Comment:
The second argument to GAP's `HashKeyBag` is actually also a `seed`. So we could have something like this:
```suggestion
  return UInt(GAPWrap.HashPermutation(GapObj(x), GapInt(h)))
```
and then adjust `HashPermutation` suitably.

And of course a coupld tests in `test/Group/` would be nice -- just compute the hash of a couple permutations etc., and e.g. create `(1,2)` twice and make sure both copies have the same hash, and different from that of `()`. Or so